docs/admin/fido/config.md

```diff
@@ -26,21 +26,21 @@ tags:
-| skipDownloadMdsEnabled                  | false                                                                            | Boolean value indicating whether the MDS download should be omitted                                     |
-| skipValidateMdsInAttestationEnabled     | false                                                                            | Boolean value indicating whether MDS validation should be omitted during attestation                    |
-| assertionOptionsGenerateEndpointEnabled | false                                                                            | Boolean value indicating whether the assertion custom endpoint (used especially in passkey) is enabled. |
+| Field named                             | Example                                                                        | Description                                                                                             |
+|-----------------------------------------|--------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------|
+| authenticatorCertsFolder                | /etc/jans/conf/fido2/authenticator_cert                                        | Authenticators certificates fodler.                                                                     |
+| mdsCertsFolder                          | /etc/jans/conf/fido2/mds/cert                                                  | MDS TOC root certificates folder.                                                                       |
+| mdsTocsFolder                           | /etc/jans/conf/fido2/mds/toc                                                   | MDS TOC files folder.                                                                                   |
+| serverMetadataFolder                    | /etc/jans/conf/fido2/server_metadata                                           | Authenticators metadata in json format. Example: virtual devices.                                       |
+| metadataUrlsProvider                    | https://mds3.fido.tools                                                        | String value to provide source of URLs with external metadata.                                          |
+| enabledFidoAlgorithms                   | ["RS256","ES256"]                                                              |                                                                                                         |
+| rp                                      | [{"id":"https://my-jans-server.jans.io","origins":["my-jans-server.jans.io"]}] | Requested party id.                                                                                     |
```

Comment:
id -> rpId, and it should be set to host "my-jans-server.jans.io"

origins need to be ["https://my-jans-server.jans.io"]